README.md

```diff
@@ -59,8 +58,16 @@ or install top level dataset by omitting `-r datasets` option and then calling
 
 You can navigate datasets you have installed in your terminal or browser,
 while fetching necessary files or installing new sub-datasets using the
-same `datalad install [FILENAMES]` command.
-
+`datalad get [FILE|DIR]` command.  DataLad will take care about
```

Comment:
This actually is about line 57: Not true anymore. `install` doesn't operate on lists of things to be installed currently. If you think this should be changed - let's discuss in hangout. But while you are at it, this should be adapted to be two calls for now.